internal/cli/deployments/connect.go

```diff
@@ -28,6 +28,11 @@ func Run(ctx context.Context, opts *options.ConnectOpts) error {
 	return opts.Connect(ctx)
 }
 
+func PostRun(opts *options.ConnectOpts) error {
```

Comment:
why not
```suggestion
func (opts *options.ConnectOpts) PostRun() error {
```

Reply:
we can't define it there in `connect.go` and I found too weird to define it in the opts file, so I followed same run convention, I tried this [approach before](https://github.com/mongodb/mongodb-atlas-cli/pull/2803/commits/a6555e4e8f4ed803390a0e344722528e90b41362) but didn't like it so kept this way

Reply:
it seems to me from a distance we may be falling into the same traps that made the setup command unmaintainable with complicated "inheritance" that's making commands hard to modify

Reply:
I think what we can do here, is to move everything from connect_opts_atlas.go and connect_opts_local.go into connect.go then, would that make sense? if so, I can do that in a next PR to avoid growing this one, I can start the draft now

Reply:
Let's open a ticket and discuss as a team if worth the effort given we plan to re write a lot of this logic

Reply:
thanks! https://jira.mongodb.org/browse/CLOUDP-239577